docs/src/content/docs/guide/language/basics.mdx

```diff
@@ -3,13 +3,13 @@
 title: Basics
 description: The very basics of Slint
 ---
-import CodeSnippetMD from '../../../components/CodeSnippetMD.astro';
+import CodeSnippetMD from '../../../../components/CodeSnippetMD.astro';
 import { Aside, LinkCard } from '@astrojs/starlight/components';
 
 <LinkCard
-  title="VsCode Quick start"
+  title="Vscode Quick start"
```

Comment:
I believe the correct spelling is „VS Code“. This is what they’re using on their site and docs, and I think we should, too.

Reply:
Thank you! It looked wrong and I could't work out what was the right way.